modules/db.js

```diff
@@ -16,9 +16,7 @@ exports.init = function() {
         // if db file doesn't exist then create it
         try {
             log.debug(`Check that db exists: ${filePath}`);
-
-            fs.accessSync(filePath, fs.R_OK);
-
+            fs.accessSync(filePath, fs.W_OK);
```

Comment:
Yeah, saw that. That looks fine to me. We could do `fs.R_OK | fs.W_OK` I suppose.